app/styles/app.scss

```diff
@@ -595,6 +596,9 @@ hr {
             }
         }
     }
+    button {
```

Comment:
Ah. For the followup PR, styles for a specific component should go in a separate file, not the main shared app.scss. This will give us a bit more isolation and support refactoring.

Move them to `styles/componentName.scss`, and `@import` into app.scss at the top.